mujoco_warp/_src/types.py

```diff
@@ -598,6 +599,7 @@ class Option:
   sdf_initpoints: int
   sdf_iterations: int
   run_collision_detection: bool  # warp only
+  ls_min_step: float  # warp only
```

Comment:
is this only used in parallel linesearch?  if so, should we call it `ls_parallel_min_step` ?

also, if it is only for parallel linesearch, let's move it underneath `ls_parallel`?  also, might want to annotate that `ls_parallel` is warp only